Makefile

```diff
@@ -1,18 +1,19 @@
 INSTALL=install
 PREFIX=/usr
 SYSCONFDIR=/etc
+PKG_CONFIG ?= pkg-config
```

Comment:
Why `?=` instead of `=`? Can you please keep the line consistent with the lines above?